go/vt/mysqlctl/replication.go

```diff
@@ -228,6 +230,38 @@ func (mysqld *Mysqld) GetServerUUID(ctx context.Context) (string, error) {
 	return conn.Conn.GetServerUUID()
 }
 
+// GetGlobalStatusVars returns the server's global status variables asked for.
+// An empty/nil variable name parameter slice means you want all of them.
+func (mysqld *Mysqld) GetGlobalStatusVars(ctx context.Context, statuses []string) (map[string]string, error) {
+	query := "SELECT variable_name, variable_value FROM performance_schema.global_status"
+	if len(statuses) != 0 {
+		// The format specifier is for any optional predicates.
+		statusBv, err := sqltypes.BuildBindVariable(statuses)
+		if err != nil {
+			return nil, err
+		}
+		query, err = sqlparser.ParseAndBind("SELECT variable_name, variable_value FROM performance_schema.global_status WHERE variable_name IN %a",
+			statusBv,
+		)
+		if err != nil {
+			return nil, err
+		}
+	}
+	qr, err := mysqld.FetchSuperQuery(ctx, query)
```

Comment:
IMO it's worth creating a `mysqld.FetchQuery` helper (doesn't have to be done here) as there should be no need to use the DBA pool for this and various other things.

Reply:
We can do that change as a follow up. Currently rushing to get this into RC